Tools/Scripts/upload-file-to-url

```diff
@@ -22,6 +22,7 @@
 # (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
 # SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
+import webkitpy
```

Comment:
will importing webkitpy automatically install packages like requests?

Reply:
It will automatically install some packages. More important, though, it sets up the autoinstaller so that importing registered packages (which `requests` is one of) will result in those packages being installed before they are used.